modules/cluster-logging-collector-log-forward-cloudwatch.adoc

```diff
@@ -0,0 +1,290 @@
+
+You can include a specific CloudWatch topic in the output or use the default. The CloudWatch output can use a TCP (insecure) or TLS (secure TCP) connection.
+
+.Procedure
+
+. Create a `Secret` YAML file similar to the following example. Use base64-encoded values for the `username` and `password` fields. For example:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  name: cw-secret
+  namespace: openshift-logging
+data:
+  aws_access_key_id: F1lAUtMIGg2LzANQkVIWVhSRFQKQ
+  aws_secret_access_key: aSEvRkovGEpPMkxZNHhWK1dwb2p0UY1kvHlNdEt6Z0VhNHBSV1J3Lgo=
+----
+
+
+. Create the secret. For example:
++
+[source,terminal]
+----
+$ oc create secret -n openshift-logging cw-secret.yaml
+----
+
+
+. Create a `ClusterLogForwarder` CR YAML file. In it, specify the name of the secret. For example:
++
+[source,yaml]
+----
+apiVersion: "logging.openshift.io/v1"
+kind: ClusterLogForwarder
+metadata:
+  name: instance <1>
+  namespace: openshift-logging <2>
+spec:
+  outputs:
+   - name: cw <3>
+     type: cloudwatch <4>
+     cloudwatch:
+       groupBy: logType <5>
+       region: us-east-2 <6>
+     secret:
+        name: cw-secret <7>
+  pipelines:
+    - name: infra-logs <8>
+      inputRefs: <9>
+        - infrastructure
+        - audit
+        - application
+      outputRefs:
+        - cw <10>
+----
+<1> The name of the `ClusterLogForwarder` CR must be `instance`.
+<2> The namespace for the `ClusterLogForwarder` CR must be `openshift-logging`.
+<3> Specify a name for the output.
+<4> Specify the `cloudwatch` type.
+<5> TBD? Optional: Specify the option CloudWatch uses to group the logs.
+<6> TBD? Optional: Specify the region or regions CloudWatch uses.
+<7> Specify the name of the secret.
+<8> Optional: Specify a name for the pipeline.
+<9> Specify which log types to forward by using the pipeline: `application,` `infrastructure`, or `audit`.
+<10> Specify the name of the output to use when forwarding logs with this pipeline.
+
+. Create the CR object:
++
+[source,terminal]
+----
+$ oc create -f <file-name>.yaml
+----
+
+The Red Hat OpenShift Logging Operator redeploys the Fluentd pods. If the pods do not redeploy, you can delete the Fluentd pods to force them to redeploy.
+
+[source,terminal]
+----
+$ oc delete pod --selector logging-infra=fluentd
+----
+
+'''
+
+.Example: How various configurations of the ClusterLogForwarder affect CloudWatch
+
+Suppose that you run an {product-title} cluster on AWS named `myusername-48` and the value of its `infrastructureName` is `myusername-48-7977k`.
+
+[source,terminal]
+----
+$ oc get Infrastructure/cluster -ojson | jq .status.infrastructureName
+"myusername-48-7977k"
+----
+
+The default project on the cluster is called `app`.
+
+[source,terminal]
+----
+$ oc project
+Using project "app" on server "https://api.myusername-48.devcluster.openshift.com:6443".
+----
+
+The default project has a UUID that you will reuse.
```

Comment:
We won't really reuse the UUID, it's more like we will encounter it again further down.

Reply:
Updated. Thanks.